.mergify.yml

```diff
@@ -0,0 +1,37 @@
+pull_request_rules:
+  - name: automatic merge
+    conditions:
+      - and: &base_checks
+        - base=master
+        - -label~=^acceptance-tests-needed|not-ready
+        - -body~=PLACEHOLDER
+        - "status-success~=static tests.*"
+        - "status-success~=unit tests.*"
+        - "status-success~=compile tests.*"
+        - "#check-failure=0"
+        - "#check-pending=0"
+        - linear-history
+      - and:
+        - "#approved-reviews-by>=2"
+        - "#changes-requested-reviews-by=0"
+        # https://doc.mergify.io/examples.html#require-all-requested-reviews-to-be-approved
+        - "#review-requested=0"
+        - "body~=Verification run: http"
```

Comment:
i would suggest to add `work-in-progress` label here https://docs.mergify.com/examples/#id9